src/test/acceptance/features/unsuccessful-application.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Unsuccessful application
+  In order to apply for the HTBHF programme
+  As a potential claimant
+  If my claim is unsuccessful I want to be shown the why my claim was unsuccessful
+
+  Scenario Outline: When a claimant applies for help to buy healthy foods and is unsuccessful they are told why
+    Given I am on the first page of the application
+    When I submit an application which returns a <status> status
```

Comment:
Can this step be renamed to `I submit an application which returns a <status> eligibility status` otherwise you might think its the HTTP status or something else.

Reply:
Done.